### ISSUE TITLE
Add `git-prs` or update `git-pr` command to pull multiple pull requests at once

### ISSUE THREAD

**Issue by ZhilkinSerg** [tj/git-extras#741]
### Description:

Please add `git-prs` or update `git-pr` command to pull multiple pull requests at once.

### Example (based on existing `git-pr`):

```BASH
#!/bin/sh

remote_default=upstream

pull_requests=$1
remote=$2

if [[ -z "${1}" ]]; then
  echo Please provide a comma separate string of pull request numbers.
  exit 0
fi

if [[ -z "${2}" ]]; then
  remote=$remote_default
fi

echo Pulling pull request numbers [$pull_requests] from remote [$remote].

for pull_request in $(echo $pull_requests | sed "s/,/ /g")
do
  git pr $pull_request $remote
done

```

### Usage:

Following command

`git prs 12345,12678,52345 upstream`

would run

```
git pr 12345 upstream
git pr 12678 upstream
git pr 52345 upstream
```

**Comment by spacewander** [comment]
@ZhilkinSerg 
Does #743 fix your requirement?

**Comment by spacewander** [comment]
Consider solved as #743 is merged.

**Comment by ZhilkinSerg** [comment]
Thanks a lot, @spacewander!
